media-libs/dav1d/dav1d-0.5.2.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="A new AV1 cross-platform decoder"
+DESCRIPTION="dav1d is a new AV1 cross-platform decoder, open-source, \
+and focused on speed and correctness. It supports all features from AV1, \
+including all subsampling and bit-depth parameters."
+HOMEPAGE="https://code.videolan.org/videolan/dav1d"
+COPYRIGHT="2018-2019, VideoLAN and dav1d authors"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://code.videolan.org/videolan/dav1d/-/archive/$portVersion/dav1d-$portVersion.tar.bz2"
+CHECKSUM_SHA256="1780199d71e62a82ca1bfca2d71da5c599c7e1edc68c72f6fc3c0c25ce1880b7"
+
+ARCHITECTURES="?x86_gcc2 x86 x86_64 ?arm ?ppc"
+
+libVersion="3.1.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	dav1d = $portVersion
+	cmd:dav1d
+	lib:libdav1d = $libVersionCompat
+	"
+REQUIRES="
+	haiku
+	"
+
+PROVIDES_devel="
+	dav1d_devel = $portVersion
+	devel:libdav1d = $libVersionCompat
+	"
+REQUIRES_devel="
+	dav1d == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:doxygen
+	cmd:gcc
+	cmd:meson >= 0.47
+	cmd:nasm >= 2.13.02
+	cmd:ninja
+	"
+
+defineDebugInfoPackage dav1d \
+	$binDir/dav1d
+
+BUILD()
+{
+	rm -f -rf build
+	mkdir build && cd build
+
+	meson ..
+	meson configure \
+		--prefix $prefix \
+		--bindir $binDir \
+		--libdir $libDir \
+		--includedir $includeDir
+	ninja $jobArgs
```

Comment:
Keep as it was:
```diff
- 	ninja $jobArgs
+ 	ninja -C build $jobArgs
```

Reply:
In the previous lines, I have already `mkdir build && cd build`, so ninja is supposed to build in the current directory.

Reply:
OK, need to have a second look for that later :)

Reply:
Can you switch to `meson build/` instead of manually creating a directory?

Reply:
Check https://github.com/haikuports/haikuports/blob/45c8bd0905c98472cb536497d4384d1f2f9583b7/dev-lang/orc/orc-0.4.30.recipe#L53

Reply:
Still needs to be changed